plugins/modules/vmware_cluster_dpm.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Swisscom (Schweiz) AG
+# Copyright: (c) 2018, Ansible Project
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vmware_cluster_dpm
+short_description: Manage Distributed Power Management (DPM) on VMware vSphere clusters
+description:
+    - Manages DRS on VMware vSphere clusters.
+    - All values and VMware object names are case sensitive.
+author:
+- Swisscom (Schweiz) AG
```

Comment:
This should be you in the form `First Last (@GitHubID)` (see [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-fields))

Reply:
This is not how our company wants to handle this

Reply:
Hi @Miclusc,

The format of the field is enforced by the some sanity checks. If you want to give credit to your company, I would just suggest to add it at the top of the file. This is an example: https://github.com/ansible-collections/community.vmware/blob/main/plugins/modules/vmware_cfg_backup.py#L4-L5

Reply:
Hi @goneri
Thx for the hint, I altered the script to fit the suggestion